src/scripts.js

```diff
@@ -32,27 +32,28 @@ const createHotel = (data) => {
 
 const startSite = () => {
   user = new User (hotel.allUsers[0]);
+  updateSite()
+};
+
+const updateSite = () => {
+  console.log(user)
   user.allBookings = hotel.findUsersBookings(user.id)
   user.divideBookingsByDate()
-  console.log(user.allBookings)
-  console.log(user)
   domUpdates.updateLeftDisplay(user);
-};
-
-
+}
 
 const updateBookings = (data) => {
-  console.log(hotel.bookings)
-  hotel.bookings = data.bookings.map(booking => booking)
-  console.log(hotel.bookings)
-}
+  hotel.bookings = data.bookings;
+};
+
+const sendBookingToApi = (date, roomNumber) => {
```

Comment:
YES!!! I have been trying to figure out how to keep all my API stuff in the apiCalls file. This is awesome!